packages/query/src/queriers/tendermint.ts

```diff
@@ -16,4 +19,12 @@ export class TendermintQuerier implements TendermintQuerierInterface {
     const res = await this.client.getStatus(req);
     return res.syncInfo!.latestBlockHeight;
   }
+
+  async broadcastTx(params: Uint8Array) {
```

Comment:
Think we've had a new method added here in a recent PR merge

Reply:
rebased